examples/complete/main.tf

```diff
@@ -270,3 +277,8 @@ resource "random_pet" "this" {
   length = 2
 }
 
+resource "aws_cloudfront_function" "example" {
+  name    = "example"
```

Comment:
Add `random_pet` resource to a name to make sure it does not exist (see any other examples in other terraform-aws-modules for more)